pkg/worker/worker.go

```diff
@@ -544,7 +544,7 @@ func (s *Worker) spawn(request *types.ContainerRequest, bundlePath string, spec
 	go s.updateContainerStatus(request)
 
 	// Set worker hostname
-	hostname := fmt.Sprintf("%s:%d", s.podAddr, defaultWorkerServerPort)
+	hostname := fmt.Sprintf("%s:%d", s.podAddr, s.runcServer.port)
```

Comment:
is there nowhere else we need to adjust?

Reply:
there is no where else to adjust for `defaultWorkerServerPort`. Other places get the worker runc server address through the containerRepo redis row